### PR TITLE
[LMS-386] Change from schoolyearkey to schoolyear in PostgreSQL

### PR DESCRIPTION
**Pull request by lalrg**

### CODE REVIEW

**Comment by stephenfuqua** [comment]
Because the existing file was a mistake, and has not been released yet, I asked Luis to modify instead of adding a new file. Not something we do often, but basically safe in this regard IMHO. @bptillman.

**Review by stephenfuqua — COMMENTED** [review]
The change looks good visually. Waiting to see the TeamCity build process work before merging.

**Comment by bptillman** [comment]
> 
> 
> Because the existing file was a mistake, and has not been released yet, I asked Luis to modify instead of adding a new file. Not something we do often, but basically safe in this regard IMHO. @bptillman.

That is fine with me, but the file should keep the structure of DROP IF EXISTS and then CREATE. 

> 
> 
> Because the existing file was a mistake, and has not been released yet, I asked Luis to modify instead of adding a new file. Not something we do often, but basically safe in this regard IMHO. @bptillman.

That is fine with me as well in this instance. One thing I did notice is that the structure of the script was changed from DROP IF EXISTS and CREATE to just a CREATE OR REPLACE VIEW and my understanding thus far of AMT was that for any Postgres view alterations we did the DROP IF EXISTS and CREATE to better handle if columns changed, since the migration scripts will throw an error like they did in this instance
`cannot change name of view column "schoolyearkey" to "schoolyear" `

File: src/EdFi.AnalyticsMiddleTier.DataStandard32/Base/PostgreSQL/0034-View-AcademicTimePeriodDim-Alter.sql [hunk 1]
```diff
@@ -3,80 +3,74 @@
...
-        AND 
-        SectionClassPeriod.SchoolId = Course.EducationOrganizationId
-        AND 
-        SectionClassPeriod.SchoolYear = CourseOffering.SchoolYear
-        AND 
-        SectionClassPeriod.SectionIdentifier=s.SectionIdentifier
-        AND 
-        SectionClassPeriod.SessionName=s.SessionName;
\ No newline at end of file
+CREATE OR REPLACE VIEW analytics.AcademicTimePeriodDim
```
Comment (stephenfuqua) [inline src/EdFi.AnalyticsMiddleTier.DataStandard32/Base/PostgreSQL/0034-View-AcademicTimePeriodDim-Alter.sql:6]: @lalrg please see @bptillman 's comment about dropping the view first.

**Review by stephenfuqua — CHANGES_REQUESTED** [review]

**Review by stephenfuqua — APPROVED** [review]

**Comment by stephenfuqua** [comment]
@bptillman can you do a final review & approval before we merge this, since it is part of the core view collection?

**Comment by bptillman** [comment]
> 
> 
> @bptillman can you do a final review & approval before we merge this, since it is part of the core view collection?

Can do

**Review by bptillman — APPROVED** [review]

**Pull request merged**